src/modules/trip/controllers/oneWayTrip-controller.js

```diff
@@ -0,0 +1,42 @@
+const { OneWayTrip } = require('../model');
+
+/**
+ * OneWayTrip Controller Class
+ */
+class OneWayTripController {
+  /**
+     * @param {Express.Request} req
+     * @param {Express.Response} res
+     *  @returns {*} created trip
+    */
+  static async createOneWayTrip(req, res) {
+    try {
+      const {
+        destination, departure, date, reason, accomodationId
+      } = req.body;
+
+      const createdBy = req.user.id;
+      // Create the new trip
+      const trip = await OneWayTrip.create({
+        departure,
+        destination,
+        date,
+        reason,
+        created_by: createdBy,
+        accomodationId,
+      });
+
+      res.status(201).json({
+        trip,
+        message: 'Trip created successfully'
+      });
+    } catch (error) {
+      console.error(error);
+      res.status(500).json({ error: 'Server error' });
+    }
```

Comment:
- [ ] Best make sure a nomad will not be able to book more than one trip in the same accommodation, that means you will validate dates incase he/she books a trip with the same accommodation to get an error

Reply:
fixed: 
<img width="959" alt="image" src="https://user-images.githubusercontent.com/60257249/228332151-31c3047f-4e0c-480d-ad31-53ed30518c32.png">